themes/digital.gov/layouts/landings/list.html

```diff
@@ -0,0 +1,59 @@
+{{ define "content" }}
+  {{ $path := "" }}
+  {{ with .File }}
+    {{ $path = .Path }}
+  {{ else }}
+    {{ $path = .Path }}
+  {{ end }}
+
+
+  <main role="main" id="main-content">
+    <header class="dg-landing__hero">
+      {{- partial "core/img-flexible.html" (dict "Site" .Site "src" $.Params.image "small" "false") -}}
+      <div class="dg-landing__hero-text">
+        <h1>{{- .Params.title -}}</h1>
+        <a href="#content-start" class="usa-button usa-button--big"
+          >Get Started
+        </a>
+      </div>
+    </header>
+
+    <article class="dg-landing__body grid-container grid-container-desktop">
+      <div data-edit-this="{{- $path -}}">
+        {{- .Content -}}
+      </div>
+      <h2 class="dg-landing__body-featured-resources">Featured Resources</h2>
+      <ul class="usa-card-group" id="content-start">
+        {{- range .Pages -}}
+          <li class="usa-card tablet:grid-col-4">
+            <div class="usa-card__container">
+              <div class="usa-card__media usa-card__media--indent">
+                <div class="usa-card__img">
+                  {{- partial "core/img-flexible.html" (dict "Site" .Site "src" .Params.image "small" "true") -}}
+                </div>
+              </div>
+              <div class="usa-card__header">
+                <h2 class="usa-card__heading">
+                  {{- .Title | markdownify -}}
+                </h2>
+              </div>
+              <div class="usa-card__body">
+                <p>{{- .Params.summary | markdownify -}}</p>
+              </div>
+              <div class="usa-card__footer">
+                <a href="{{- .Permalink | absURL -}}" class="usa-button"
+                  >Read Guide</a
```

Comment:
Can we add text so that this is unique and not so repetitive?

See the community cards on https://digital.gov/communities/.

<img width="991" alt="image" src="https://github.com/GSA/digitalgov.gov/assets/3385219/c1f0ff6e-f71b-48e2-89fb-cdc0c5a4f0e0">